src/axolotl/utils/schemas/validation.py

```
@@ -0,0 +1,1073 @@
+                )
+        return data
+
+    @model_validator(mode="before")
+    @classmethod
+    def check_lora_8bit(cls, data):
+        if (
+            data.get("lora_mlp_kernel")
+            or data.get("lora_qkv_kernel")
+            or data.get("lora_o_kernel")
+        ):
+            if data.get("adapter") == "lora" and data.get("load_in_8bit"):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)